pilot/pkg/kube/inject/webhook.go

```diff
@@ -210,7 +211,11 @@ func (wh *Webhook) Run(stop <-chan struct{}) {
 				log.Errorf("update error: %v", err)
 				break
 			}
-
+			caCertPEM, err := ioutil.ReadFile(wh.certFile)
```

Comment:
Does the server's cert file (`wh.certFile`) include the CA bundle?

Reply:
Ah, that was a mistake - fixed.